python/tests/tests.py

```diff
@@ -40,16 +41,13 @@ def id(self):
 
     def roundtrip(self, val, mutable=True):
         enc = self.bser_mod.dumps(val)
-        print "# %s  -->  %s" % (val, enc.encode('hex'))
+        print "# %s  -->  %s" % (repr(val), enc.encode('hex'))
```

Comment:
There is a default `repr` implementation, but in any case `val` should never be a bser tuple.